pkg/daemon/ceph/client/mirror.go

```diff
@@ -136,7 +136,24 @@ func disablePoolMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, p
 	return nil
 }
 
-func removeClusterPeer(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, peerUUID string) error {
+// DisablePoolImageMirroring turns off mirroring on a pool image
+func DisablePoolImageMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, poolName, imageName string) error {
```

Comment:
what about the images created with rbd radosnamespace?

Reply:
After discussion, not removing the mirrored images anymore.
When support for cleanup annotation is implemented, then images can be deleted if annotation is present